### PR TITLE
Update error message  master

### PR DESCRIPTION
**Pull request by ManishPrajapati-GoFynd**

### CODE REVIEW

File: src/lib/CommandError.ts [hunk 1]
```diff
@@ -37,15 +37,19 @@ export const ErrorCodes = {
...
     code: 'FDK-0009'
   },
   NO_APP_FOUND: {
-    message: "Sales channel not found",
+    message: "No sales channel found, Please create a sales channel under development/live account. For more details, refer to the documentation link:https://platform.fyndx1.de/help/docs/manage-website/create-website",
     code: 'FDK-00010'
   },
   NO_THEME_FOUND: {
-    message: "Theme not found",
+    message: "No themes are created, Please create/add a theme under the sales channel. For more details, refer to the documentation link: https://partners.fyndx1.de/help/docs/partners/themes/vuejs/overview",
```
Comment (vivek-gofynd) [inline src/lib/CommandError.ts]: All links should be dynamic here it should also support sandbox. @manishprajapati52

Comment (ManishPrajapati-GoFynd) [reply src/lib/CommandError.ts]: **Done**

File: src/lib/Theme.ts [hunk 2]
```diff
@@ -177,7 +179,7 @@ export default class Theme {
         const companyListOptions = {};
         if (!companyList.items.length) {
             throw new CommandError(
-                ErrorCodes.NO_COMPANY_FOUND.message,
+                ErrorCodes.NO_COMPANY_FOUND.message(company_type, getBaseURL().replace('api', 'partners')),
```
Comment (vivek-gofynd) [inline src/lib/Theme.ts]: can it be moved into `src/lib/CommandError.ts` @manishprajapati52

Comment (ManishPrajapati-GoFynd) [reply src/lib/Theme.ts]: Added common function to get projects domain.

File: src/lib/Theme.ts [hunk 3]
```diff
@@ -204,7 +206,7 @@ export default class Theme {
             }
         });
         if (!applicationList.data.items.length) {
-            throw new CommandError(ErrorCodes.NO_APP_FOUND.message, ErrorCodes.NO_APP_FOUND.code);
+            throw new CommandError(ErrorCodes.NO_APP_FOUND.message(getProjectURL('partners')), ErrorCodes.NO_APP_FOUND.code);
```
Comment (vivek-gofynd) [inline src/lib/Theme.ts]: Why do we need to pass `getProjectURL('partners')` here as an argument? @manishprajapati52

Comment (ManishPrajapati-GoFynd) [reply src/lib/Theme.ts]: @vivek-gofynd we have resolve that in new function.

**Review by vivek-gofynd — APPROVED** [review]

**Pull request merged**